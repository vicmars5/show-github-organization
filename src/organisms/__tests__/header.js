import React from "react"
import renderer from "react-test-renderer"

import Header from "../header"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header
        name="MyOrgName"
        description="Elit laborum fugiat impedit aut quaerat"
        avatarURL="#"
        url="https://github.com"
      />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
