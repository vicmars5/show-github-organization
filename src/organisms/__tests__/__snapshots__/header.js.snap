// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="border-b border-gray-400 bg-white p-3"
>
  <div
    className="container mx-auto py-4 flex"
  >
    <img
      alt="MyOrgName"
      className="w-20 h-20 mr-2"
      src={null}
    />
    <div>
      <a
        className="text-color-link"
        href="https://github.com"
      >
        <h1
          className="font-bold text-xl"
        >
          MyOrgName
        </h1>
      </a>
      <p>
        Elit laborum fugiat impedit aut quaerat
      </p>
    </div>
  </div>
</header>
`;
